### ISSUE TITLE
Microcore Quagga

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-registry#26]

**Comment by adosztal** [comment]
Instead of this, I'll add the frrouter container, which is a quagga fork.

**Comment by adosztal** [comment]
There was no official VM so I built one. PR: #339

**Comment by ehlers** [comment]
It's correct, that FRR needs to run as a privileged container, but all container in GNS3 run privileged (to be able to set the IP address). So that's no issue.

In my understanding even a privileged container doesn't change the routing table of the host. Otherwise the default route in a docker container should change the default route of the host, what's not the case. I've tested other static routes and they also don't change the host routes.

So I did some basic tests with OSPF in a FRR container (https://hub.docker.com/r/ehlers/frr/) and that works flawless. There may be bad side effects, when running FRR in a container, but I haven't found one during my quick test. I'm very  interested to hear of any issue running FRR in a container.
